packages/components/tests/dummy/app/templates/components/tabs.hbs

```diff
@@ -5,8 +5,15 @@
 <section>
   <h3 class="dummy-h3" id="overview"><a href="#overview" class="dummy-link-section">§</a> Overview</h3>
   <p class="dummy-paragraph">
-    Tabs allow users to move between different views within the same context and at the same level of hierarchy, ie. UI
-    vs CLI, macOS vs Windows vs Linux, etc.
+    Tabs allow users to
+    <strong>move between different views within the same context</strong>
+    and at the same level of hierarchy, ie. UI vs. CLI, macOS vs. Windows vs. Linux, etc.
+  </p>
+  <p class="dummy-paragraph">
+    Tabs are not intended for primary navigation. For this reason, we chose to use HTML
+    <code class="dummy-code">button</code>
+    elements for activating individual tabs vs. links. If your use case includes a usage of a Tabs-like component for
+    primary navigation, consider reaching out to your designer and working with them to refactor the design if feasible.
```

Comment:
This language is feeling geared more toward engineers. What do you think about making it more generic? 

```suggestion
   navigation, consider using other navigation patterns, such as sidebar, links, or breadcrumbs.
```

Reply:
Ok thanks

Reply:
@heatherlarsen @KristinLBradley not for this ticket,  but probably in a follow-up (maybe when we review the documentation for the new website): I think we need to clarify what we mean with "Tabs are not intended for navigation". This is confusing ourselves too (me, surely!). I _think_ we're conflating _navigation_ with _URL changing_, which are not necessarily the same thing (and I suspect readers would do the same). So we have to be more clear with what users/designers/developers are **not** supposed to do or are not supposed to use the tabs for.

Reply:
I think it would be very helpful to clarify that difference more. I can do a 2nd follow on PR to add more explanation around that after we discuss more.